documentation/src/main/asciidoc/topics/config_examples/server_ldap_realm_ad.xml

```diff
@@ -0,0 +1,17 @@
+<security xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
where is this example used?

Reply:
I had removed it by accident. Added to the security_server.adoc